documentation/src/main/asciidoc/reference/search-dsl-predicate.asciidoc

```diff
@@ -784,11 +784,78 @@ but can be <<search-dsl-predicate-common-boost,boosted>>,
 either on a per-field basis with a call to `.boost(...)` just after `.field(...)`/`.fields(...)`
 or for the whole predicate with a call to `.boost(...)` after `.matchingAny(...)` or `.matchingAll(...)`.
 
+[[search-dsl-predicate-and]]
+// TODO YB: find references to _combining_queries and add references to _anding_queries
+== [[_anding_queries]] `and`: combine predicates as a conjunction
```

Comment:
I'm not sure where this `_anding_queries` anchor comes from? Unless there was such a (generated) anchor in previous versions of the documentation, you don't need that.

Reply:
> (Looks like section anchors are not much used either, but that's another discussion...)

They are useful when answering user questions and pointing to the documentation.

Reply:
Ah!  There's a link when we hover over titles!